src/stargate/cudos-signingstargateclient.ts

```diff
@@ -156,4 +157,33 @@ export class CudosSigningStargateClient extends SigningStargateClient {
         const { msg, fee } = await this.nftModule.msgRevokeNft(addressToRevoke, denomId, tokenId, sender, '', gasPrice, gasMultiplier, memo);
         return this.signAndBroadcast(sender, [msg], fee, memo);
     }
+
+    public async msgMultisend(
+        sender: {
+            address: string,
+            coins: Coin[]
+        }[],
+        recipients: {
+            address: string,
+            coins: Coin[]
+        }[],
+        gasPrice: GasPrice,
+        gasMultiplier = 1.3,
```

Comment:
could we not hardcode this multiplier here, but use a constant instead, or a config file?